relay-config/src/redis.rs

```diff
@@ -208,7 +208,7 @@ pub enum RedisConfigRef<'a> {
 /// Helper struct bundling connections and options for the various Redis pools.
 #[allow(clippy::large_enum_variant)]
 #[derive(Clone, Debug)]
-pub enum RedisPoolConfigs<'a> {
+pub enum RedisConfigsRef<'a> {
```

Comment:
Changed name since now it's not anymore a pool and it was not a good name also before.